composer.json

```diff
@@ -17,7 +17,7 @@
         "codeception/codeception": "4.0.x-dev | ^4.0"
     },
     "require-dev": {
-        "pda/pheanstalk": "~3.0",
+        "pda/pheanstalk": "> 3",
```

Comment:
Lower boundary is not clear and upper boundary is not set.
I think that `"^3.0 | ^4.0"` would be better.

Reply:
P.S. Please update comment in "suggest" which says thay v4 is not supported.